modules/structs/settings.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package structs
+
+// GeneralRepoSettings contains general repo settings exposed by api
+type GeneralRepoSettings struct {
+	MirrorsDisabled  bool `json:"mirrors_disabled"`
+	HTTPGitDisabled  bool `json:"http_git_disabled"`
+	MaxCreationLimit int  `json:"max_creation_limit"`
```

Comment:
This can be different per-user, should we really expose the default? I don't see much reason to and it might be used improperly.

Reply:
🤔 you are right - the best is to remove it for this pull - getting user specific settings will need a new endpoint i think